types/node/index.d.ts

```diff
@@ -6553,6 +6554,7 @@ declare module "util" {
         export function isArrayBuffer(object: any): object is ArrayBuffer;
         export function isAsyncFunction(object: any): boolean;
         export function isBooleanObject(object: any): object is Boolean;
+        export function isBoxedPrimitive(object: any): object is (Number | Boolean | String | Symbol /* BigInt */);
```

Comment:
This is causing a problem for me when targeting es5. I'm getting `<root>/node_modules/@types/node/index.d.ts(7436,88): error TS2304: Cannot find name 'Symbol'.`.

The path points to `utils.d.ts` in later versions.

I've tried including the `es2015.symbol` lib, but that doesn't appear to solve the problem.

I have had to revert to earlier types for now. 

The other references in this file to Symbol are all `symbol` (lowercase). Are there any workarounds, or should this be changed to refer to `symbol`?